<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabInterface">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Interface</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Appearance</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>33</y>
              <width>601</width>
              <height>271</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="cbCloseButtonHidesApp">
                 <property name="text">
                  <string>Close Window hides application</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbShowToolbar">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Show toolbar</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbShowStatusBar">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Show statusbar</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbShowPackageTooltip">
                 <property name="text">
                  <string>Show package tooltip</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbStartHidden">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Start with hidden window</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QCheckBox" name="cbHighlightItems">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>400</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Limit highlighted search items at</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="sbHighlightItems">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>70</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>0</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <number>100</number>
                   </property>
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>9</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="lblPrivilege">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>450</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Privilege escalation tool:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbPrivilege">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>0</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="lblFontSize">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>270</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Adjust font size:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="sliderFont">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="5" column="0">
              <spacer name="verticalSpacer_7">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>498</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabPkgtools">
        <attribute name="title">
         <string>Package tools</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>498</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Backend</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="topMargin">
               <number>10</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="rbPkgTools">
                  <property name="text">
                   <string>Pkgtools</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblPkgtools">
                  <property name="text">
                   <string>Slackware's official tool to manage packages</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="indent">
                   <number>30</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbSpkg">
                  <property name="text">
                   <string>Spkg</string>
                  </property>
                  <property name="shortcut">
                   <string>Ctrl+S</string>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblSpkg">
                  <property name="text">
                   <string>A fast and robust tool for Slackware package management</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::AutoText</enum>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="indent">
                   <number>30</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QCheckBox" name="cbUseSilentOuput">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Use silent ouput (responsive interface)</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="lblObs">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>This option can be changed at any time</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabUpdater">
        <attribute name="title">
         <string>Slackware mirror list</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblMirror">
           <property name="text">
            <string>Choose a Slackware mirror for patch download</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QTableWidget" name="twMirror">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cbAutoCheckUpdates">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Automatically check for new patches at startup</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
